plugin/data_masking/src/plugin.cc

```diff
@@ -67,3 +74,17 @@ mysql_declare_plugin(data_masking){
     NULL,                            // config options
     0,                               // flags
 } mysql_declare_plugin_end;
+
+// Returns true and populates the msg with an error if the module has not been initialized
+bool data_masking_validate_status(char *msg, size_t msg_len) {
```

Comment:
rename this as "is_plugin_inited()"
return true if the plugin is inited, else false.

Reply:
on second thought, I feel this should be an optional review comment ;)

Reply:
so upto you :)